master/manifests/calico-policy-only.yaml

```diff
@@ -6,4 +6,5 @@ datastore: kdd
 typha:
   enabled: true
 ipam: host-local
+network: none
```

Comment:
This is dual-homed in master and v3.7 - you'll need to change the "getting-started/kubernetes/intallation/..." one as well

Reply:
Some of the manifests already appear to be fixed so I did not find any other manifests that needed fixing in `v3.7` and `master`.
https://github.com/projectcalico/calico/blob/master/master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/policy-only/1.7/calico.yaml#L8
https://github.com/projectcalico/calico/blob/master/v3.7/getting-started/kubernetes/installation/hosted/kubernetes-datastore/policy-only/1.7/calico.yaml#L8